### PR TITLE
Style cleanup

### PR DESCRIPTION
**Pull request by wrefgtzweve**
Should make the repo mostly conform with cfc's guidelines.

### CODE REVIEW

File: lua/autorun/client/boom_headshot.lua [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
 end
 
-hook.Add("Initialize","CreateBlood.TTT", function()
-	CreateSpriteMaterials()
-end)
+hook.Add( "Initialize", "CreateBlood.TTT", function()
+    CreateSpriteMaterials()
+end )
 
 hook.Remove( "CreateClientsideRagdoll", "FWKZT.Headshot.CreateCRagdolls" )
```
Comment (plally) [inline lua/autorun/client/boom_headshot.lua]: Could get rid of these hook.Removes if you want.

**Comment by wrefgtzweve** [comment]
Funny thing though, i've noticed that this also works clientsided on sandbox. Could take a look if this can be added to cfc3 as non-default optional? Just an idea.

File: lua/autorun/client/boom_headshot.lua [hunk 2]
```diff
@@ -1,34 +1,38 @@
...
-		util.Effect( 'headshot', ED )
-	end
+    if entity.WasHeadshotDeath and entity:WasHeadshotDeath() then
+        local head = ragdoll:LookupBone( "valvebiped.bip01_head1" )
+        if not head then return end
+        local pos = ragdoll:GetBonePosition( head )
+        local ragHead = ragdoll:LookupBone( "valvebiped.bip01_head1" )
+        if not ragHead then return end
+        ragdoll:ManipulateBoneScale( ragHead, Vector( 0, 0, 0 ) )
+        local eD = EffectData()
```
Comment (plally) [inline lua/autorun/client/boom_headshot.lua]:
```suggestion
        ragdoll:ManipulateBoneScale( ragHead, Vector( 0, 0, 0 ) )

        local eD = EffectData()
```
I think the newline they had there was good, helps break it up a little bit.

File: lua/autorun/server/boom_headshot.lua [hunk 3]
```diff
@@ -1,85 +1,68 @@
...
+    if not IsValid( Ply.server_ragdoll ) then return end
+    if not Ply.was_headshot then return end
+    if not IsValid( Attacker ) or not Attacker:IsPlayer() then return end
+    if not IsValid( Attacker:GetActiveWeapon() ) then return end
+    if Ply.IsGhost and Ply:IsGhost() then return end
 
-	local Head = Ply:LookupBone('valvebiped.bip01_head1')
-	if !Head then return end
-	local Pos = Ply:GetBonePosition( Head )
+    if Ply.OwnedBlackMarketItems and Ply.OwnedBlackMarketItems[CAT_EQUIPMENT] then
```
Comment (plally) [inline lua/autorun/server/boom_headshot.lua]: shouldnt this be `Ply.OwnedBlackMarketItems[CAT_EQUIPMENT] == "fiber_helmet"`

Comment (wrefgtzweve) [reply lua/autorun/server/boom_headshot.lua]: Good catch

File: lua/autorun/client/boom_headshot.lua (hunk 2 above)
Comment (wrefgtzweve) [reply lua/autorun/client/boom_headshot.lua]: Yeah, i used my formatter to do the bulk work, i usually add spaces in when im modifying the code but i'll do a run through it an space it properly

**Comment by wrefgtzweve** [comment]
Applied the suggestions

**Pull request closed without merging**